lib/improver/orographic_enhancement.py

```diff
@@ -566,14 +543,9 @@ def process(self, temperature, humidity, pressure, uwind, vwind,
                 UKPP domain grid
 
         Returns:
-            (tuple): tuple containing:
-                **orogenh** (iris.cube.Cube):
-                    Precipitation enhancement due to orography in mm/h on the
-                    1 km Transverse Mercator UKPP grid domain
-                **orogenh_standard_grid** (iris.cube.Cube):
-                    Precipitation enhancement due to orography in mm/h on the
-                    UK standard grid, padded with masked np.nans where outside
-                    the UKPP domain
+            iris.cube.Cube:
+                Precipitation enhancement due to orography in mm/h on the
+                1 km Transverse Mercator UKPP grid domain
```

Comment:
No it doesn't ... could you please replace the `in mm/h` with `in m/s` or `in SI units` here as the internals of the plugin have changed.  I'm also not happy with how we've specified the grid in the docstring as this is necessary for the UK IMPROVER suite, but is not actually enforced.

Reply:
Done, and removed the part about grid size.